students/rmart300/mailroom/mailroom.py

```diff
@@ -0,0 +1,65 @@
+from datetime import date
+
+#dictionary to hold donors and list of donation amounts
+donation_dict = {}

```

Comment:
looks like you're ahead of the game since we will need to use `dict` in the next part of the mailroom 🙂